### PR TITLE
Escape contact names in several HTML snippets/jQuery insert contexts

### LINKED ISSUE #9538: Possible problem with nicknames that hold a blind attack in their screenname
When people from Friendica get a request to connect from a specially crafted nickname it messes up the template:

https://f.haeder.net/display/adbba6c6-145f-b2d5-36b5-dbf024802469

In this case the name is being crafted with:

The nick belongs to me. It is a part of a simple test to see if it is possible to insert a blind attack to a site. Might need to be filtered properly or fixed in the template checks

### PR DESCRIPTION
**Pull request by MrPetovan**
Fixes #9538

Hotfix PR: https://github.com/friendica/friendica/pull/9539

Thanks to @hansw-nlo for his inoffensive "blind attack sniffer" username that helped uncover this security hole.

### CODE REVIEW

File: mod/ping.php [hunk 1]
```diff
@@ -337,28 +337,16 @@ function ping_init(App $a)
...
-		usort($notifs, $sort_function);
-
-		if (DBA::isResult($notifs)) {
-			foreach ($notifs as $notif) {
-				$contact = Contact::getByURL($notif['url'], false, ['micro', 'id', 'avatar']);
-				$notif['photo'] = Contact::getMicro($contact, $notif['photo']);
-
-				$local_time = DateTimeFormat::local($notif['date']);
-
-				$notifications[] = [
```
Comment (annando) [inline mod/ping.php:349]: Why is this removed?

Comment (MrPetovan) [reply mod/ping.php:349]: It's been replaced by a much simpler `array_walk` call since only two keys of the array items have to be updated.

Comment (annando) [reply mod/ping.php:349]: Well, some PHP is still black magic to me.
